index.html

```diff
@@ -87,15 +195,22 @@
 
 				<section>
 					<h2>Example Blogs and Bloggers</h2>
+          <h4>We like these:</h4>
 					<ul>
-						<li><a href="https://blog.womenhackfornonprofits.com">Official WHFNP Blog</a></li>
-						<li><a href="http://blog.lilianakastilio.co.uk/">Lili Kastilio</a></li>
-						<li><a href="http://blog.tanyapowell.co.uk/">Tanya Powell</a></li>
-						<li><a href="http://echesters.co.uk/">Elizabeth Chesters</a></li>
-						<li><a href="https://medium.com/@krissygoround">Krissy Rabino</a></li>
+            <li><a href="http://www.bennadel.com">Ben Nadel</a></li>
+            <li><a href="http://www.missgeeky.com">Melinda Seckington</a></li>
 						<li><a href="http://matthewjamestaylor.com/blog/">Matthew James Taylor</a></li>
 						<li><a href="https://medium.com/cool-code-pal">CSS Perverts</a></li>
```

Comment:
If I have time at lunch I'll have a google of over bloggers to add. 
Apart from that let's merge this thang 💃